dll/win32/wininet/lang/wininet_Pl.rc

```diff
@@ -27,17 +27,17 @@ STRINGTABLE
   IDS_CERT_DATE_INVALID "Data certyfikatu jest nieprawidłowa."
   IDS_CERT_CN_INVALID   "Nazwa na certyfikacie nie pasuje do witryny."
   IDS_CERT_ERRORS       "Istnieje co najmniej jeden nieokreślony problem bezpieczeństwa z tym certyfikatem."
-  IDS_CERT_SUBJECT      "Subject"
-  IDS_CERT_ISSUER       "Issuer"
-  IDS_CERT_EFFECTIVE    "Effective Date"
-  IDS_CERT_EXPIRATION   "Expiration Date"
-  IDS_CERT_PROTOCOL     "Security Protocol"
-  IDS_CERT_SIGNATURE    "Signature Type"
-  IDS_CERT_ENCRYPTION   "Encryption Type"
-  IDS_CERT_PRIVACY      "Privacy Strength"
-  IDS_CERT_HIGH         "High"
-  IDS_CERT_LOW          "Low"
-  IDS_CERT_BITS         "bits"
+  IDS_CERT_SUBJECT      "Podmiot"
+  IDS_CERT_ISSUER       "Wydawca"
+  IDS_CERT_EFFECTIVE    "Data Rozpoczęcia"
+  IDS_CERT_EXPIRATION   "Data Wygaśnięcia"
+  IDS_CERT_PROTOCOL     "Protokół Bezpieczeństwa"
+  IDS_CERT_SIGNATURE    "Rodzaj Podpisu"
+  IDS_CERT_ENCRYPTION   "Rodzaj Szyfrowania"
+  IDS_CERT_PRIVACY      "Poziom Prywatności"
+  IDS_CERT_HIGH         "Wysoka"
+  IDS_CERT_LOW          "Niska"
+  IDS_CERT_BITS         "bit"
```

Comment:
Why it's in singular form?

Reply:
IMO it makes more sense this way as we're not doing declination

Reply:
Just in case of doubt, this string is being used in generating certificate information string. It can be tested on a machine with Polish localization with this API test: https://github.com/reactos/reactos/blob/97a61654c5d8cd986b30ff60b3112d3b28fc9041/modules/rostests/winetests/wininet/http.c#L6474

Reply:
Usually we just go with "-ów" as this is correct in most cases. Also this is what the winetest shows on Polish Windows 8.1.

Reply:
OK I can do that